### ISSUE TITLE
Review: toggle for "approached reviewer outside of Janeway"

### ISSUE THREAD

**Issue by Astronaut-resting** [openlibhums/janeway#3179]
Issue: At OLHJ we have a policy that editors should contact reviewers outside of Janeway first, using their institutional email, to ask permission before sending an official request for review via the system (more polite, less likely to be treated as junk). It is our Special Collection Editors who approach reviewers, and our job as editors to oversee that they are making progress and staying on track with the collection. However, because they approach reviewers outside of Janeway first, and that process can be lengthy, we have no way of knowing whether they are making progress with their collection until that official Janeway request goes out.

Solution: a toggle button or similar, where the Special Collection editor can select that a reviewer as been "tentatively approached" or "initial email contact made". This would allow us to have a truer picture of the status of the collection. I am not sure whether the name of the potential reviewer should also be noted down at this point. 

**Describe alternatives you've considered**
Just leaving it and trusting that it is happening behind the scenes. Or emailing them to check in and make sure, which can feel like nagging. 

**Additional context**
Happy to further discuss the advantages/drawbacks of this or any other solutions 

Katherine

**Comment by ajrbyers** [comment]
So rather than an active review you could select a Reviewer and mark them as “approached”? Let’s add this to the agenda for the Ed/tech meeting.

**Comment by AmeliePlaysKath** [comment]
Yes, something like that. Though might need to make it clear that the approach has been tentative and informal so as to distinguish it from an official approach via Janeway. 
Also, the issue with doing this by selecting a Reviewer through Janeway is that it would mean adding their details in advance and we have been asked not to add their details into Janeway before asking their permission informally via email because of GDPR. Which complicates things. Let's discuss further!

**Comment by ajrbyers** [comment]
> Yes, something like that. Though might need to make it clear that the approach has been tentative and informal so as to distinguish it from an official approach via Janeway.
> Also, the issue with doing this by selecting a Reviewer through Janeway is that it would mean adding their details in advance and we have been asked not to add their details into Janeway before asking their permission informally via email because of GDPR. Which complicates things. Let's discuss further!

GDPR is fun and games. There are ways around it like deleting the details if they don’t accept. Let’s have a chat and come up with some ideas.

**Comment by ajrbyers** [comment]
BRF: We discussed this and think the workflow described is quite specific to OLHJ and not something we have time to build. 

There are alternatives for recording these including notes and the upcoming discussions field.
